<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cwSurveyEditorMainWindow</class>
 <widget class="QMainWindow" name="cwSurveyEditorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cave-vapor-where - Version: Super-ultra-alpha-3000 - Don't redistribute without permission :D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QDeclarativeView" name="DeclarativeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionSurvexExport"/>
     <addaction name="actionCompass"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="actionSurvexImport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSurvexImport">
   <property name="text">
    <string>Import Survex</string>
   </property>
  </action>
  <action name="actionReloadQML">
   <property name="text">
    <string>Reload QML</string>
   </property>
  </action>
  <action name="actionSurvexExport">
   <property name="text">
    <string>Survex (.svx)</string>
   </property>
  </action>
  <action name="actionCompass">
   <property name="text">
    <string>Compass (.plt)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>cwSurveyEditorMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
